### ISSUE TITLE
Add periodic (hourly) chime option to gadgetbridge

### ISSUE THREAD

**Issue by b16r05** [Freeyourgadget/Gadgetbridge#1004]
#### Before opening an issue please confirm the following:
- [Y] I have read the [wiki](https://github.com/Freeyourgadget/Gadgetbridge/wiki), and I didn't find a solution to my problem / an answer to my question.
- [Y] I have searched the [issues](https://github.com/Freeyourgadget/Gadgetbridge/issues), and I didn't find a solution to my problem / an answer to my question.
- [Y] If you upload an image or other content, please make sure you have read and understood the [github policies and terms of services](https://help.github.com/articles/github-terms-of-service/#1-responsibility-for-user-generated-content)

#### Your Gadgetbridge version is: latest always

*New issues about already solved/documented topics could be closed without further comments. Same for too generic or incomplete reports.*

########################################

Hello,
MiBand 2 can remind you through gadgetbridge settings if you are sitting too long, to get up and walk around. The same signal (buzz) could be used as an periodic (hourly) chime to "ring", like a CASIO watch does. Quiet hours could also be set for night time.
 
An android app on Fdroid does this very well... source code is available here: https://github.com/raatmarien/chibe
Thanks!

**Comment by danielegobbetti** [comment]
Is the app you linked also emitting notifications? Are they intercepted by gadgetbridge?

**Comment by b16r05** [comment]
@danielegobbetti 
Yes, (the app) chibe does have the option to show notifications in android. I don't think that they are intercepted by gadgetbridge. 

One option I saw was to put the logic straight into gadgetbridge (and watch) and use it independently of the app. Another way is to use the app and simply intercept app notification and "buzz". But, the latter would require continuous bluetooth connection to a phone.

**Comment by danielegobbetti** [comment]
There is no way we can add this feature directly on devices that do not support it natively. Want we could do is to integrate the logic within gadgetbridge (but you would need to be connected). If there is an external open source app that can handle this scenario and we can forward its notification, I see no need for it, though.

**Comment by NotAFile** [comment]
One issue with using an external application is that there is no way to set a symbol or text or custom vibration for it

**Comment by LuccoJ** [comment]
Alarms are much more intrusive though... Gadgetbridge already has the ability to use one or more alarm slots in a device to send "sticky" calendar events, that will cause an alarm even if the watch happens to be disconnected at that time. But, it's a full-blown alarm that has to be dismissed by the user... not great for an hourly chime...

**Comment by b16r05** [comment]
@LuccoJ , thank you for that. And I agree, full-blown (native) alarm is not a good option. And, the other option as @danielegobbetti mentioned is to have continuous bluetooth connectivity and only trigger a gadget BUZZ through a notification. My original intention was to have something similar to a CASIO watch BEEP BEEP, instead BUZZ BUZZ vibration every hour (<-- for which there is no native gadget support). Anyway, thank you all for looking into this.
